<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetDialog</class>
 <widget class="QDialog" name="AssetDialog">
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>470</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Create New Asset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="assetLabel">
       <property name="text">
        <string>Asset Name (max 3 characters) :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="assetLineEdit">
       <property name="toolTip">
        <string>Alphanumeric only</string>
       </property>
       <property name="maxLength">
        <number>3</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="catGroup">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>Category</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="catBG">
          <property name="text">
           <string>BG</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">catBtnGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="catCH">
          <property name="text">
           <string>CH</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">catBtnGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="catFX">
          <property name="text">
           <string>FX</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">catBtnGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="catProps">
          <property name="text">
           <string>Props</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">catBtnGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="catVehicles">
          <property name="text">
           <string>Vehicles</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">catBtnGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="descGroup">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Description (disabled)</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="descName">
          <property name="text">
           <string>Full Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="descNameLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="descHint">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remember to write meaningful Full Name for easy identification. E.g.:&lt;/p&gt;&lt;p&gt;HSE - House&lt;br/&gt;PAU - Pope's Aura Flame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>descNameLineEdit</zorder>
       <zorder>descName</zorder>
       <zorder>descHint</zorder>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="previewLabel">
     <property name="text">
      <string>*Preview must be checked to enable Create</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="previewGroup">
     <property name="title">
      <string>Preview</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="previewText">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>previewGroup</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>previewLabel</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="catBtnGroup"/>
 </buttongroups>
</ui>
